community/bolt/src/main/java/org/neo4j/bolt/v1/runtime/internal/CypherAdapterStream.java

```diff
@@ -72,7 +80,7 @@ public boolean visit( Result.ResultRow row ) throws Exception
         } );
 
         QueryExecutionType qt = delegate.getQueryExecutionType();
-        visitor.addMetadata( "type", queryTypeCode( qt.queryType() ) );
+        visitor.addMetadata( "type", queryType );
```

Comment:
Are we still sending `type` in the `SUCCESS` for `PULL_ALL`? Shall we just remove it? 
Then we also need to modify in drivers to save `type` info when they got the `SUCCESS` for `RUN`